nsis/src/nsis/format/NsisStrings.java

```diff
@@ -0,0 +1,24 @@
+package nsis.format;
+
+import ghidra.app.util.bin.BinaryReader;
+
+public class NsisStrings {
+	private int sectionLength;
+
+	/**
+	 * When creating a NsisStrings object, the pointer of the reader is advanced to
+	 * the end of the strings section.
+	 * 
+	 * @param reader
+	 * @param stringsSectionOffset
+	 * @param languageSectionOffset
+	 */
+	public NsisStrings(BinaryReader reader, int stringsSectionOffset, int languageSectionOffset) {
```

Comment:
I don't think NsisStrings object should be aware that the strings section is followed by the language section. You can use generic param names here: sectionStartOffset, sectionEndOffset.